vars/ploigosWorkflowStandard.groovy

```diff
@@ -631,6 +631,7 @@ def call(Map paramsMap) {
                                     set -eu -o pipefail
 
                                     source ${HOME}/${WORKFLOW_WORKER_VENV_NAME}/bin/activate
+                                    export SONAR_SCANNER_OPTS='-Djavax.net.ssl.trustStore=/etc/pki/java/cacerts'
```

Comment:
@redhat-snelson id rather we do this in the PSR rather then hard code it into jenkins so then its portable to any other CI tool we use.